addons/udes_priorities/models/priority_groups.py

```diff
@@ -1,3 +1,4 @@
+from typing import Tuple
```

Comment:
Why the new import?

Reply:
beat me to it, it's not used? :O

Reply:
I removed it :P I also noticed that there was a missing depends on the compute function